bubbly/builtin/release.go

```diff
@@ -1,18 +1,32 @@
 package builtin
 
 func ReleaseStatusByStages(release Release) string {
-
 	for _, stage := range release.ReleaseStage {
-		for _, criteria := range stage.ReleaseCriteria {
-			if len(criteria.ReleaseEntry) == 0 {
-				return "PENDING"
-			}
-			for _, entry := range criteria.ReleaseEntry {
-				if !entry.Result {
-					return "BLOCKED"
-				}
-			}
+		status := ReleaseStageStatus(stage)
+		if status != "READY" {
```

Comment:
Maybe `iota` with a `String()` method for these string constants? [Example](https://yourbasic.org/golang/iota/)

Reply:
Yes I want to do that once I know what the values are and where to put them

Reply:
Done